README.Rmd

```diff
@@ -18,6 +18,7 @@ knitr::opts_chunk$set(
 [![MIT license](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/epiforecasts/covidregionaldata/blob/master/LICENSE.md/)   [![GitHub contributors](https://img.shields.io/github/contributors/epiforecasts/covidregionaldata)](https://github.com/epiforecasts/covidregionaldata/graphs/contributors)  [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-yellow.svg)](https://makeapullrequest.com/) [![GitHub commits](https://img.shields.io/github/commits-since/epiforecasts/covidregionaldata/v0.9.1.svg?color=orange)](https://GitHub.com/epiforecasts/covidregionaldata/commit/master/) [![DOI](https://zenodo.org/badge/271601189.svg)](https://zenodo.org/badge/latestdoi/271601189) [![status](https://joss.theoj.org/papers/dd6f7acdae3b7136a3ac373ce9a0655c/status.svg)](https://joss.theoj.org/papers/dd6f7acdae3b7136a3ac373ce9a0655c)
 
 Interface to subnational and national level COVID-19 data sourced from both official sources, such as Public Health England in the UK, and from other Covid-19 data collections, including the World Health Organisation (WHO), European Centre for Disease Prevention and Control (ECDC), John Hopkins University (JHU), Google Open Data and others. Covid-19 data is cleaned and processed from their raw format in an open and transparent way, allowing users to scrutinise and extend our methods. For all countries supported, this includes a daily time-series of cases. Wherever available data is provided on deaths, hospitalisations, and tests. National level data is also supported using a range of data sources as well as line list data and links to intervention data sets.
+ This package is designed for people who wan't access to standardised Covid-19 data from 'official' sources.
```

Comment:
yer I would just drop this basically...

Reply:
I think that Covid-19 data iss cleaned and process.... captures the use case. I don't think we need to say that users who want standardised data should use this package - that is just everyone.

Reply:
I have gone with a slight variation which is less on the nose but highlights what our package does differently to others offering an interface to covid-19 data.